### PR TITLE
DEV: Run automated tests in GH actions

### LINKED ISSUE #9: Consider enabling GitHub actions?
It would probably be good to bring this repo in line with other Enthought open source projects, and move away from Travis to GitHub actions. I'm also interested in potentially building and publishing wheels using GitHub actions, though that may be a bit more tricky (see https://github.com/enthought/vpsearch/pull/8). 

Would it be possible to enable GitHub actions for this repo? I can put in a quick PR to update the CI.

### PR DESCRIPTION
**Pull request by jvkersch**
fixes #9

### CODE REVIEW

File: .github/workflows/run-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Tests
+
+on:
+  pull_request
```
Comment (jvkersch) [inline .github/workflows/run-tests.yaml:4]: I _think_ this should work -- when I set the event trigger to "push" I get a clean build on Linux and macOS. I'm not sure how to test things when the trigger is set to `pull_request`. I feel like I'm overlooking something here...

**Comment by jvkersch** [comment]
This PR also removes all Travis configuration -- someone will have to turn off Travis builds at some point.

File: .github/workflows/run-tests.yaml (hunk 1 above)
Comment (mdickinson) [reply .github/workflows/run-tests.yaml:4]: The GitHub Actions documentation isn't totally clear here; I think there may be an issue with creating a *new* workflow from a fork. I was expecting to see an option to approve running the workflow, but that's absent.

**Comment by mdickinson** [comment]
I recreated this using exactly the same commits in PR #11, and the workflow kicked in and passed on that PR. Given that, I'll merge this one and close #11.

The Travis CI run has been removed as a required status check for the master branch protection.

**Review by mdickinson — APPROVED** [review]
LGTM

**Comment by jvkersch** [comment]
Thanks @mdickinson, much appreciated! I didn't think about the possibility that a workflow created from a fork wouldn't be picked up automatically, but that makes sense. I'm glad you got things to work.

**Pull request merged**